src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala

```diff
@@ -381,7 +381,7 @@ case object BasicApp extends App {
       environmentVariables := Map.empty,
       startScriptLocation := "/rp-start",
       secrets := Set.empty,
-      reactiveLibVersion := "0.5.0",
+      reactiveLibVersion := "0.7.0",
```

Comment:
Not released yet, hence this build failing on Travis. Won't merge until it's released of course.